Només a libmng-1.0.10/: .ilist
Només a libmng-1.0.10/: libmng.1.dll
Només a libmng-1.0.10/: libmng.a
Només a libmng-1.0.10/: libmng_callback_xs.o
Només a libmng-1.0.10/: libmng_chunk_descr.o
Només a libmng-1.0.10/: libmng_chunk_io.o
Només a libmng-1.0.10/: libmng_chunk_prc.o
Només a libmng-1.0.10/: libmng_chunk_xs.o
Només a libmng-1.0.10/: libmng_cms.o
Només a libmng-1.0.10/: libmng_display.o
Només a libmng-1.0.10/: libmng_dither.o
Només a libmng-1.0.10/: libmng_error.o
Només a libmng-1.0.10/: libmng_filter.o
Només a libmng-1.0.10/: libmng_hlapi.o
Només a libmng-1.0.10/: libmng_jpeg.o
Només a libmng-1.0.10/: libmng_object_prc.o
Només a libmng-1.0.10/: libmng_pixels.o
Només a libmng-1.0.10/: libmng_prop_xs.o
Només a libmng-1.0.10/: libmng_read.o
Només a libmng-1.0.10/: libmng_trace.o
Només a libmng-1.0.10/: libmng_write.o
Només a libmng-1.0.10/: libmng_zlib.o
diff -ru libmng-1.0.10-orig//makefiles/makefile.mingw libmng-1.0.10//makefiles/makefile.mingw
--- libmng-1.0.10-orig//makefiles/makefile.mingw	2005-01-30 11:28:00.000000000 +0100
+++ libmng-1.0.10//makefiles/makefile.mingw	2010-11-07 17:16:18.000000000 +0100
@@ -97,12 +97,12 @@
 	$(AR2) $@
 
 install : $(LIBMNG_A)
-	$(MKDIR)                 $(INSTALL_PREFIX)include	
-	$(COPY)  libmng.h        $(INSTALL_PREFIX)include
-	$(COPY)  libmng_conf.h   $(INSTALL_PREFIX)include
-	$(COPY)  libmng_types.h  $(INSTALL_PREFIX)include
-	$(MKDIR)                 $(INSTALL_PREFIX)lib
-	$(COPY)  $(LIBMNG_A)     $(INSTALL_PREFIX)lib
+	$(MKDIR)                 $(INSTALL_PREFIX)/include	
+	$(COPY)  libmng.h        $(INSTALL_PREFIX)/include
+	$(COPY)  libmng_conf.h   $(INSTALL_PREFIX)/include
+	$(COPY)  libmng_types.h  $(INSTALL_PREFIX)/include
+	$(MKDIR)                 $(INSTALL_PREFIX)/lib
+	$(COPY)  $(LIBMNG_A)     $(INSTALL_PREFIX)/lib
 
 clean:
 	$(RM) *.o
diff -ru libmng-1.0.10-orig//makefiles/makefile.mingwdll libmng-1.0.10//makefiles/makefile.mingwdll
--- libmng-1.0.10-orig//makefiles/makefile.mingwdll	2005-01-30 11:28:00.000000000 +0100
+++ libmng-1.0.10//makefiles/makefile.mingwdll	2010-11-07 17:17:31.000000000 +0100
@@ -46,6 +46,8 @@
 
 # compiler
 CC=gcc
+AR=ar rc
+RANLIB= ranlib
 
 ALIGN=
 # for i386:
@@ -53,7 +55,6 @@
 
 CFLAGS=$(ZLIBINC) $(JPEGINC) $(LCMSINC) -Wall -O3 -funroll-loops $(OPTIONS) $(ALIGN) -s
 LDFLAGS=-L. -lmng $(ZLIBLIB) $(JPEGLIB) $(LCMSLIB) -lm -s
-
 INCPATH=$(prefix)/include
 LIBPATH=$(prefix)/lib
 
@@ -85,18 +86,22 @@
 
 all: $(LIBMNG_A)
 
-$(LIBMNG_A): $(LIBMNG_DLL)
+$(LIBMNG_A) : $(OBJS)
+	$(RM) $@
+	$(AR) $@ $(OBJS)
+	$(RANLIB) $@
 
-$(LIBMNG_DLL) : $(OBJS)
-	dllwrap --implib=$(LIBMNG_A) --dllname=$(LIBMNG_DLL) $(OBJS) $(LDFLAGS)
+$(LIBMNG_DLL) : $(LIBMNG_A)
+	$(CC) --shared -o $(LIBMNG_DLL) $(LIBMNG_A) $(LDFLAGS)
    
-install : $(LIBMNG_A)
-	$(MKDIR)                 $(INSTALL_PREFIX)include	
-	$(COPY)  libmng.h        $(INSTALL_PREFIX)include
-	$(COPY)  libmng_conf.h   $(INSTALL_PREFIX)include
-	$(COPY)  libmng_types.h  $(INSTALL_PREFIX)include
-	$(MKDIR)                 $(INSTALL_PREFIX)lib
-	$(COPY)  $(LIBMNG_A)     $(INSTALL_PREFIX)lib
+install : $(LIBMNG_A) $(LIBMNG_DLL)
+	$(MKDIR)                 $(INSTALL_PREFIX)/include	
+	$(COPY)  libmng.h        $(INSTALL_PREFIX)/include
+	$(COPY)  libmng_conf.h   $(INSTALL_PREFIX)/include
+	$(COPY)  libmng_types.h  $(INSTALL_PREFIX)/include
+	$(MKDIR)                 $(INSTALL_PREFIX)/lib
+	$(COPY)  $(LIBMNG_A)     $(INSTALL_PREFIX)/lib
+	$(COPY)  $(LIBMNG_DLL)     $(INSTALL_PREFIX)/lib
 
 clean:
 	$(RM) *.o
Només a libmng-1.0.10//makefiles: .makefile.mingwdll.swp
Només a libmng-1.0.10//makefiles: .makefile.mingw.swp
diff -ru libmng-1.0.10-orig//README libmng-1.0.10//README
--- libmng-1.0.10-orig//README	2007-07-09 13:24:38.000000000 +0200
+++ libmng-1.0.10//README	2010-11-07 17:05:26.000000000 +0100
@@ -1,6 +1,8 @@
 libmng 1.0.10
 -------------
 
+
+
 Added provisional ANG and anIM support, and made some minor bugfixes.
 
 libmng 1.0.9
